src/openassetio-core/managerAPI/ManagerInterface.cpp

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2013-2022 The Foundry Visionmongers Ltd
-
 #include <openassetio/managerAPI/ManagerInterface.hpp>
+#include <stdexcept>
```

Comment:
[consider][minor] I've tried to do the PEP8-like thing of putting includes in the order of system, library, project (with cpplint additionally enforcing C headers before C++). To satisfy clang-format, you can put a blank line between groups of headers, then they're only sorted within the group.

Reply:
Do you think we need them in `C++` where the return type is already documented, and there is little ambiguity in the method itself? I could see merit if were talking about ownership etc... but as its a Ptr?

Reply:
Totally, fixed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/471/commits/3c0bce34058b7c29fea90a8b16e8c0c7b1c554fd).